### ISSUE TITLE
Need script for OCIO Windows

### ISSUE THREAD

**Issue by meshula** [meshula/mkvfx#9]

**Comment by digitaldestructo** [comment]
Hey Nick! I have OCIO and OIIO built for msvcp80-120. Let me know if you wanna take a look at the build scripts and convert it to your build system. -J

**Comment by meshula** [comment]
Yes please, that would be sweeeeeet!

**Comment by digitaldestructo** [comment]
OK - Give me a couple of days - I just realized my OCIO is cross linking to debug DLLs at the moment, so I'll have to sort that out. Also, we can discuss some of the changes I made to the CMAKES for OIIO. Mostly to support multiple msvcp versions of DLLs in the same folder, but also to support compiling the Python modules against different Python versions. I'll send it to your personal email.

**Comment by meshula** [comment]
I'm starting to modify the scripts to build windows to bin and bin_d, lib and lib_d. I had kind of forgotten about how much of a PITA that is :\

**Comment by digitaldestructo** [comment]
Yeah haha - I had to touch almost every cmake file OIIO just to output DLL/LIBS with a custom name like "*_vc90, *_vc110, etc".

I emailed all the info to build to your personal address. Try to get those going and feel free to give a call if you get stuck.

**Comment by meshula** [comment]
Sounds like a pull request to the OIIO team is also in order so that in the end modifying the cmake files locally isn't necessary!
